client/me/reauth-required/two-factor-actions.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import React, { useCallback } from 'react';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEventWithClientId } from 'calypso/state/analytics/actions';
+import './two-factor-actions.scss';
+
+interface TwoFactorActionsProps {
+	isAuthenticatorSupported: boolean;
+	isSecurityKeySupported: boolean;
+	isSmsSupported: boolean;
+	isSmsAllowed: boolean;
+	onChange: ( authType: string ) => void;
+	twoFactorAuthType: string;
+}
+
+export function TwoFactorActions( {
+	isAuthenticatorSupported,
+	isSecurityKeySupported,
+	isSmsSupported,
+	isSmsAllowed,
+	onChange,
+	twoFactorAuthType,
+}: TwoFactorActionsProps ) {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const recordButtonClicked = useCallback(
+		( event: React.MouseEvent< HTMLButtonElement > ) => {
+			event.preventDefault();
```

Comment:
This is superfluous, there is no default action to prevent. The button is not inside a form and is of `type="button"`, not `type="submit"`.

Reply:
> This is superfluous, there is no default action to prevent. The button is not inside a form and is of `type="button"`, not `type="submit"`.

Removed in 44844689ce.